### PR TITLE
GP2HUB-74 gp2 google tag manager

### PR DESCRIPTION
**Pull request by AkosuaA**

### CODE REVIEW

File: .github/workflows/reusable-build.yml [hunk 1]
```diff
@@ -192,6 +192,7 @@ jobs:
           REACT_APP_GP2_AUTH0_DOMAIN: ${{ steps.setup.outputs.gp2-auth0-domain }}
           REACT_APP_GP2_SENTRY_DSN: ${{ steps.setup.outputs.react-app-gp2-sentry-dsn }}
           REACT_APP_GTM_CONTAINER_ID: ${{ steps.setup.outputs.react-app-gtm-container-id }}
```
Comment (markcsims) [inline .github/workflows/reusable-build.yml:194]: I think this should be renamed to REACT_APP_CRN_GTM_CONTAINER_ID to reflect that it is CRN only.

File: .github/actions/setup-environment/action.yml [hunk 2]
```diff
@@ -233,6 +233,9 @@ outputs:
   react-app-gtm-container-id:
```
Comment (markcsims) [inline .github/actions/setup-environment/action.yml:233]: I think this should be renamed to react-app-crn-gtm-container-id to reflect that it is CRN only.

**Review by markcsims — CHANGES_REQUESTED** [review]

File: .github/workflows/reusable-build.yml (hunk 1 above)
Comment (AkosuaA) [reply .github/workflows/reusable-build.yml]: updated

File: .github/actions/setup-environment/action.yml (hunk 2 above)
Comment (AkosuaA) [reply .github/actions/setup-environment/action.yml]: updated

**Review by markcsims — APPROVED** [review]

**Pull request merged**